realis.go

```diff
@@ -859,6 +864,73 @@ func (r *realisClient) AbortJobUpdate(
 	return resp, nil
 }
 
+//Pause Job Update. UpdateID is returned from StartJobUpdate or the Aurora web UI.
+func (r *realisClient) PauseJobUpdate(updateKey *aurora.JobUpdateKey, message string) (*aurora.Response, error) {
+	var resp *aurora.Response
+	var clientErr error
+	retryErr := ExponentialBackoff(*r.config.backoff, func() (bool, error) {
+		resp, clientErr = r.thriftCallHelper(func() (*aurora.Response, error) {
+			return r.client.PauseJobUpdate(updateKey, message)
+		})
+
+		if clientErr != nil {
+			return false, clientErr
+		}
+
+		return true, nil
+	})
+	if retryErr != nil {
+		return nil, errors.Wrap(clientErr, retryErr.Error()+": Error sending PauseJobUpdate command to Aurora Scheduler")
```

Comment:
If clientErr becomes nil here, we'll incorrectly return a nil error. retryErr will become clientErr if clientErr is not nil as per line 877 so we can get away with just wrapping retryErr.

Reply:
@rdelval would there be a case when clientErr is nil while retryErr is not?

Reply:
Only remote possibility is that we get a TimeoutErr from the exponential backoff without the client getting set to err. I don't know if that code path is even possible, but it's hard to tell from the two closures being used. I had changed this in my PR that's on hold now. I think I will make it a point to simplify our  two closures maybe even combining them into one. As you had mentioned to me offline, this code is confusing.

Reply:
sounds good @rdelval .. it is updated.. take care of this.. thanks.